main.tf

```diff
@@ -1,3 +1,3 @@
 provider "aws" {
-  region = var.region
+  region = "us-east-1"
```

Comment:
Your backend is in `eu-north-1` but your infrastructure is in `us-east-1` it's better to keep state file closer to infrastructure. 

Otherwise, it will take significant time to refresh terraform state when your env grows